include/clover_field_order.h

```diff
@@ -467,41 +460,41 @@ namespace quda {
 	 * @param[in] dim Which dimension we are taking the norm of (dummy for clover)
 	 * @return L1 norm
 	 */
-	__host__ double norm1(int dim=-1, bool global=true) const {
+	__host__ double norm1(int =-1, bool global=true) const {
```

Comment:
Is the `=-1` still necessary here?

Reply:
Yes, it need to have a default value for the case where we have no arguments.